test/fixtures/templates/quickstart/cis_benchmark.yaml

```diff
@@ -306,7 +306,7 @@
         Handler: index.lambda_handler
         MemorySize: 1024
         Role: !GetAtt MasterConfigRole.Arn
-        Runtime: python2.7
+        Runtime: python3.8
```

Comment:
Thinking the other way where test results are updated with old runtime results:
https://github.com/aws-cloudformation/cfn-lint/pull/2083 would solve this forever whereas updating new runtimes in test templates would go on forever